src/App.js

```diff
@@ -1,34 +0,0 @@
-import { Fragment, useState } from "react";
-
-import Title from "./components/Layout/Title";
-import Words from "./components/Words/Words";
-import Bravo from "./components/Bravo/Bravo";
-
-function App() {
-  const [modalIsShown, setModalIsShown] = useState(false);
```

Comment:
I think it's slightly more conventional to write `bool` variable names like this as `isModalShown` and `setIsModalShown`